Modules/QtWidgets/src/QmitkIOUtil.cpp

```diff
@@ -39,6 +39,40 @@ See LICENSE.txt or http://www.mitk.org for details.
 
 #include <algorithm>
 
+namespace
+{
+#if defined(_WIN32) || defined(_WIN64)
+    QString s_InvalidFilenameCharacters("<>:\"/\\|?*");
+#else
+    QString s_InvalidFilenameCharacters("/");
+#endif
+
+  // Return 'true' if the specified filename contains characters not accepted by the file system.
+  //
+  // This test is not exhaustive but just excluding the most common problems.
+  bool IsIllegalFilename(const QString &fullFilename)
+  {
+    QFileInfo fi(fullFilename);
+    auto filename = fi.fileName();
+
+    for (auto &ch : s_InvalidFilenameCharacters)
+    {
+      if (filename.contains(ch))
+      {
+        return true;
+      }
+    }
+
+    if (filename.startsWith(' ' || filename.endsWith(' ')))
```

Comment:
Should be noticed in function documentation.

Reply:
Implemented, see above.